src/components/cart/ModalConfirm/ModalConfirm.tsx

```diff
@@ -0,0 +1,43 @@
+import { Box, Button, Modal, Typography } from "@mui/material";
+
+import ThemeWrapper from "@/components/ThemeWrapper";
+
+import styles from "./ModalConfirm.styles";
+import constants from "./ModalConfirm.constants";
+
+interface ModalConfirmProps {
+  open: boolean;
+  setOpen: (value: boolean) => void;
+  setCartItems: (value: string | null) => void;
+}
+
+const ModalConfirm = ({ open, setOpen, setCartItems }: ModalConfirmProps) => {
+  const yesButton = () => {
+    if (typeof window !== "undefined") {
```

Comment:
Why it's not a client component, I don't think u need a modal being rendered on your server side